cpmpy/solvers/minizinc.py

```diff
@@ -572,10 +584,20 @@ def zero_based(array):
             return txt
 
         # rest: global constraints
-        elif expr.name.endswith('circuit'):  # circuit, subcircuit
+        elif expr.name == 'circuit': 
             # minizinc is offset 1, which can be problematic here...
             args_str = ["{}+1".format(self._convert_expression(e)) for e in expr.args]
 
+        elif expr.name == 'subcircuit': 
```

Comment:
Euh, this function should return a Minizinc string, but it returns nothing here?

Reply:
Here I do the exact same as circuit, which is relying on the return at the complete end of `_convert_expression`. I could copy it to the subcircuit case (and at the same time also for circuit?).